src/components/table-shadow/expandable/row/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Component, Element, Event, EventEmitter, h, Host, Prop, VNode, Watch } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-table-shadow-expandable-row",
+	styleUrl: "style.scss",
+	shadow: true,
+})
+export class SmoothlyTableShadowExpandableRow {
+	private div?: HTMLDivElement
+	@Prop({ mutable: true, reflect: true }) open = false
+	@Event() smoothlyTableExpandableRowChange: EventEmitter<boolean>
+	@Element() el: HTMLElement
+
+	clickHandler(event: MouseEvent): void {
+		;(this.div && event.composedPath().includes(this.div)) || (this.open = !this.open)
+	}
+	@Watch("open")
+	openChange() {
+		this.smoothlyTableExpandableRowChange.emit(this.open)
+	}
+
+	render(): VNode | VNode[] {
+		return (
+			<Host onClick={(e: MouseEvent) => this.clickHandler(e)}>
+				<div class="content">
+					<slot />
+				</div>
```

Comment:
content is not a fitting name for this element.
suggestions:
`row`
`summary`
Also maybe all classes should be prefixed with `"smoothly-"`?
Or maybe we should add `part` on both summary and detail this instead, like `part="summary"`.